examples/parallelReduce-api-counter/index.rsh

```diff
@@ -0,0 +1,31 @@
+'reach 0.1';
+
+export const main = Reach.App(() => {
+  const A = Participant('Admin', {
+    max: UInt,
+    launched: Fun([Contract], Null),
+  });
+  const B = API({
+    countUp: Fun([], UInt),
+  });
+  init();
+  A.only(() => {
+    const max = declassify(interact.max);
+  });
+  A.publish(max);
+  A.interact.launched(getContract());
+
+  const [count] = parallelReduce([0])
```

Comment:
FYI, you don't need to return a tuple. You can just have `const count = parallelReduce(0)....`